### PR TITLE
Add 'Timeout' section

### PR DESCRIPTION
**Pull request by henrik**
Moved it to a new document along with the stand-up text (which isn't quite accurate anymore).

- [x] KP
- [x] HN
- [ ] JK
- [ ] TS
- [x] AR

### CODE REVIEW

File: process/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+![Barsoom](http://barsoom.se/barsoom.png)
+
+# Process conventions
+
+
+## Timeout
+
+We've agreed to a "timeout" convention, where you can make a "T" with your hands (or say "timeout", in an audio conference).
+
+This is intended as a shorthand to move us along – since we've all agreed on it and made it conventional, the idea is that it's instantly understood and respected, without taking offense at the brevity.
```
Comment (Lavinia) [inline process/README.md]: This sentence is a bit too long for me to actually grasp what it says.

File: process/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+In a stand-up or other meeting, it means "enough detail, let's get back on track".
+
+For a conversation taking place in our work room, it means "would you please end this conversation or take it elsewhere".
+
+
+## Stand-up
+
+We agreed on following these stand-up conventions
```
Comment (Lavinia) [inline process/README.md]: Maybe 'we agree' instead of agreed, signaling that this is something we commit to and not just agreed on?

Comment (henrik) [inline process/README.md]: Good point. Note that this was just moved from another file, though – it's not up to date in other respects, so maybe we could do that as a separate change Some Day?

File: process/README.md (hunk 1 above)
Comment (henrik) [inline process/README.md]: Alright, will try to improve it! Thanks.

Comment (henrik) [inline process/README.md]: Better? https://github.com/barsoom/devbook/pull/42/commits/73b7f1833207ea83dc7cf05d7f7f39b243f1485c

**Comment by henrik** [comment]
Will be two weeks old tomorrow morning so I'll just merge this now. @joakimk @tskogberg If you disagree, let us know.

**Comment by henrik** [comment]
Merged manually.

**Pull request closed without merging**